### ISSUE TITLE
Problem on Map duplication

### ISSUE THREAD

**Issue by killianweid** [morehawes/Waymark#24]
When duplicating a Waymark Map texts written in the Leaflet popup are not good encoded. For example this text : 
```
Intérêt: ★★★★★☆☆☆☆☆
Commentaire: Aucun

Liens divers: Aucun

Coopération: Aucune
```
Is duplicated like this : 
```
Intu00e9ru00eat: u2605u2605u2605u2605u2605u2606u2606u2606u2606u2606
Commentaire: Aucun

n

Liens pratiques: Aucunu00a0

n

Coopu00e9ration: Aucune
```

**Comment by morehawes** [comment]
Hi @killianweid,

Thanks for bringing this to my attention. I have just released version 0.9.28.7 of the plugin, which I believe fixes this issue 🙂

Please do let me know if this is not the case for you.

[GitHub Stars greatly appreciated!](https://github.com/morehawes/Waymark/stargazers)

Cheers,

Joe

**Comment by killianweid** [comment]
Hello @morehawes, thanks for you fast correction ! It works also on my side.

Best regards,
Killian
